nifi-nar-bundles/nifi-gcp-bundle/nifi-gcp-processors/src/test/java/org/apache/nifi/processors/gcp/bigquery/BigQueryUtilsTest.java

```diff
@@ -0,0 +1,50 @@
+package org.apache.nifi.processors.gcp.bigquery;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import com.google.cloud.bigquery.Field;
+import com.google.cloud.bigquery.LegacySQLTypeName;
+import com.google.cloud.bigquery.Schema;
+import com.google.cloud.bigquery.Field.Mode;
+
+public class BigQueryUtilsTest {
+
+    @Test
+    public void can_create_a_simple_boolean_schema() throws IOException {
+        final String jsonRead = new String(
+                Files.readAllBytes(Paths.get("src/test/resources/schemas/simple_boolean_schema.json"))
+        );
+        Field booleanField = Field.newBuilder("boolean", LegacySQLTypeName.BOOLEAN).setMode(Mode.NULLABLE).build();
+        Schema expected = Schema.of(booleanField);
+        Assert.assertEquals(BigQueryUtils.schemaFromString(jsonRead), expected);

```

Comment:
`assert*` methods expect the `expected` value as their first argument, and the `actual` as the second.
It would work in this way too, only the test failure messages would be misleading, because the expected/actual would be swapped.
Please check the other two test methods too.